docs/docsite/rst/community/development_process.rst

```diff
@@ -300,44 +299,15 @@ You can find more example changelog fragments in the `changelog directory <https
 
 After you have written the changelog fragment for your PR, commit the file and include it with the pull request.
 
-.. _changelogs_how_to_format_j2_roles_playbooks:
+.. _changelogs_how_to_format_playbooks:
```

Comment:
was a search done by any chance to see if there are any referenced to the old slug?

Reply:
It was only referenced from the two other paragraphs I changed above.